src/routes/(inner)/components/input-chips/+page.svelte

```diff
@@ -114,6 +124,34 @@
 				</svelte:fragment>
 			</DocsPreview>
 		</section>
+		<section class="space-y-4">
+			<h2>Custom Hooks</h2>
```

Comment:
since it's only one 'custom hook' for now, i wasn't sure if this title was the best, but i think it's good or if/when we want to add more.